### PR TITLE
Forte static page updates

### PR DESCRIPTION
**Pull request by echang9**

### CODE REVIEW

File: app/assets/javascripts/components/footer.es6.jsx [hunk 1]
```diff
@@ -4,7 +4,7 @@ class Footer extends React.Component {
     return (
       <div className="footer">
         <div className="footer__links-container">
-          <span>© 2016 Forte Academy, Inc.</span>
+          <span>© 2017 Forte Academy, Inc.</span>
           <span>    ·    </span>
```
Comment (eric-ls) [inline app/assets/javascripts/components/footer.es6.jsx]: why is there all this space here?

File: app/assets/javascripts/components/static/home_page.es6.jsx [hunk 2]
```diff
@@ -88,6 +88,52 @@ class HomePage extends React.Component {
                 <a className="home__link" href={RouteConstants.staticPages.involvement}>Read more</a>
               </div>
             </div>
+            <div className="testimonial__line-break">
+            </div>
```
Comment (eric-ls) [inline app/assets/javascripts/components/static/home_page.es6.jsx]: nit: can you put this all on one line?
`<div className="testimonial__line-break"></div>`

File: app/assets/javascripts/components/footer.es6.jsx (hunk 1 above)
Comment (echang9) [inline app/assets/javascripts/components/footer.es6.jsx]: it's used for spacing on the footer apparently :o

Comment (eric-ls) [inline app/assets/javascripts/components/footer.es6.jsx]: it doesn't matter too much, but HTML will reduce everything to just 1 space lol

**Pull request merged**